src/targets/npm.ts

```diff
@@ -209,6 +209,16 @@ export class NpmTarget extends BaseTarget {
       // Disable output buffering because NPM/Yarn can ask us for one-time passwords
       return spawnProcess(bin, args, spawnOptions, {
         showStdout: true,
+      }).catch(error => {
+        // When publishing a list of packages fails for only one package,
+        // you can never fix this by re-running publish because it will then _always_ fail due to existing versions.
```

Comment:
an admin can check the box in publish that this target has completed -- this will skip the failing check

I'd prefer the existing approach rather than changing how craft behaves here

Reply:
But if a single step publishes multiple packages, and one of them already exists, this will not work (it does not work right now) - you can then only skip `npm` overall.

Reply:
It doesn't matter as that is the right (safest) approach to have by default. You may have out of sync packages published etc. and Craft cannot know that unless you explicitly tell it.

Reply:
@mydea in such a case I'd recommend publishing an entirely new version and yanking the partially released one.  version numbers are ~free